src/config/index.js

```diff
@@ -185,6 +186,7 @@ let config = {
   components,
 };
 
+config = applySlateConfiguration(config);
```

Comment:
ups, it's a typo, right?

Reply:
@tiberiuichim the default is slate, but the defaultBlockType remains in "text" in the config. How could that be? What is the thing that forces it? Also, it's important that we honour this setting, in the use case that someone wants to remain in "draftJS" world in Plone 6 until a migration is figured out.